api/src/services/users/users.test.ts

```diff
@@ -42,6 +112,11 @@ describe('users', () => {
   })
 
   scenario('updates a user', async (scenario: StandardScenario) => {
+    mockCurrentUser({
```

Comment:
This and `delete` are using `user` under the hood, so they follow the same validation (which I think they should 😉 ). I tested explicitly as admin here to allow the update.